contrib/testgen/README.md

```diff
@@ -2,7 +2,9 @@
 
 Utilities to generate test vectors for the data-driven Bitcoin tests.
 
-Usage:
+To use inside a scripted-diff (or just execute directly):
 
-    PYTHONPATH=../../test/functional/test_framework ./gen_key_io_test_vectors.py valid 50 > ../../src/test/data/key_io_valid.json
-    PYTHONPATH=../../test/functional/test_framework ./gen_key_io_test_vectors.py invalid 50 > ../../src/test/data/key_io_invalid.json
+    -BEGIN VERIFY SCRIPT-
```

Comment:
The `-BEGIN VERIFY SCRIPT-` bit is confusing.

Reply:
fixed